src/linter/messages.ts

```diff
@@ -276,6 +281,23 @@ export const MESSAGE_INFO = {
 		details: ({details}: {details?: string}) => details,
 	},
 
+	[MESSAGE.HTML_IN_XML]: {
+		severity: LintMessageSeverity.Error,
+		ruleId: RULES["no-deprecated-api"],
+
+		message: () => `Usage of native HTML in XML Views/Fragments is deprecated`,
+		details: () => `{@link topic:be54950cae1041f59d4aa97a6bade2d8 Using Native HTML in XML Views (deprecated)}`,
+	},
+
+	[MESSAGE.LIB_INIT_API_VERSION]: {
+		severity: LintMessageSeverity.Error,
+		ruleId: RULES["no-deprecated-api"],
+
+		message: ({libInitFunction}: {libInitFunction: string}) =>
+			`Deprecated call to ${libInitFunction}(). Use parameter {apiVersion: 2} instead`,
```

Comment:
```suggestion
			`Deprecated call to ${libInitFunction}(). Use the {apiVersion: 2} parameter instead`,
```

Reply:
Done in https://github.com/SAP/ui5-linter/pull/328/commits/fb76b91b6e38873bffd691d02d0c0cfe4d2bb904